CHANGES.md

```diff
@@ -308,6 +308,9 @@ Unreleased
 - Switch to the stag formatting API to be compatible with OCaml 5.0 (#5350,
   @emillon).
 
+- Disable warning for deprecated Toploop functions used in dune files written in
+  OCaml syntax. Restores 4.02 compatibility. (#5381, @nojb)
+
```

Comment:
That should go in a 2.9.3 release entry, if you rebase on main the 2.9.2 entry has been updated now. Sorry for annoying with this.

Reply:
Done!

Reply:
Thanks a lot!